.github/workflows/create-crd-update-pr-in-longhorn-repo.yml

```
@@ -0,0 +1,52 @@
+    - name: Update crds.yaml and manifests
+      shell: bash
+      run: |
+        curl -L https://github.com/longhorn/longhorn-manager/raw/master/k8s/crds.yaml -o chart/templates/crds.yaml
+        bash scripts/generate-longhorn-yaml.sh
+
+    - name: Create Pull Request
+      id: cpr
+      uses: peter-evans/create-pull-request@v7
+      with:
+        token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+        branch: "update-image-tags-${{ inputs.branch }}"
```

actionlint (1.7.4)

property "branch" is not defined in object type {}

(expression)